contrib/gi-docgen/template.py

```diff
@@ -0,0 +1,28 @@
+pkgname = "gi-docgen"
+pkgver = "2023.3"
+pkgrel = 0
+build_style = "meson"
+hostmakedepends = [
+    "meson",
+    "pkgconf",
+    "python-packaging",
```

Comment:
this is depends=

Reply:
also put all these python things in _deps = and then make/depends = list(_deps) since it's the same list

Reply:
also hostmakedepends i think instead of makdepends